tests/e2e/vsan_stretched_cluster_utils.go

```diff
@@ -986,3 +1009,43 @@ func waitForFullSyncToFinish(client clientset.Interface, ctx context.Context,
 	})
 	return waitErr
 }
+
+// checkForEventWithMessage fetches events list of the given object name and checkes for
+// specified message and returns true if expected error message found
+func checkForEventWithMessage(client clientset.Interface, namespace string,
```

Comment:
in that case make the wait function call this method, so that we can consolidate the code in one place